cmd/root.go

```diff
@@ -34,20 +37,19 @@ var rootCmd = &cobra.Command{
 		}
 
 		// config to define the types of files to process
-		types := []string{"pull_requests", "issues", "issue_events"}
+		types := []string{"pull_requests", "pull_request_reviews", "pull_request_review_comments", "pull_request_review_threads", "commit_comments"}
 
 		archivePath, _ := cmd.Flags().GetString("migration-archive")
-
-		err = commitremap.ProcessFiles(archivePath, types, commitMap)
+		workers, _ := cmd.Flags().GetInt("number-of-threads")
```

Comment:
@pmartindev should we check on a limit of maximum number of workers we should have ?

Reply:
Yep good idea! Maybe 50 max? I'm open to suggestions. The only risk I can think of with too many is the total memory allocation

Reply:
@pmartindev yeah that's what I was thinking going over the memory allocation and maybe causing some other issues. 50 sounds good to me, good to at least put an upper bound limit 😄

Reply:
Upper bound changed to 50